packages/loader/driver-utils/src/parallelRequests.ts

```diff
@@ -449,8 +460,6 @@ async function getSingleOpBatch(
 		} catch (error) {
 			const canRetry = canRetryOnError(error);
 
-			lastSuccessTime = undefined;
```

Comment:
Is this change intentional? Feels like it changes semantics of this code

Reply:
Yes and it does - I couldn't come up with a reason that an error should permit more attempts on getting a non-empty response, or why it would even relate to it, so I opted to remove it (in most cases it will have no effect, in the only cases where it has an effect it will just be that we still give up after 30 seconds of trying)